test_runner/src/main/kotlin/ftl/ios/xctest/common/InstallParseBinaries.kt

```diff
@@ -0,0 +1,13 @@
+package ftl.ios.xctest.common
+
+import ftl.config.FtlConstants
+import ftl.util.copyBinaryResource
+
+internal val installBinaries by lazy {
+    if (!FtlConstants.isMacOS) {
```

Comment:
Will it work properly on Windows 🤔 ?

Reply:
Not sure. It's refactoring ;P

Reply:
probably it should be 
`if (!FtlConstants.isMacOS && !isWindows)`
but if this just moving to new file, you could leave it